src/index.js

```diff
@@ -58,6 +58,10 @@ const _init = (options) => {
 
   snippet(options);
 
+  if (ready) {
```

Comment:
At first blush folks reviewing this code my perceive this as a test of state ("is it ready?") rather than a test for the existence of a function.

I think the code will be more readable if either:

You explicitly declare that you're testing for the existence of something

`if (ready !== undefined)`

or you change the parameter name to `readyCallback`

or both!

WDYT?

Reply:
> or you change the parameter name to `readyCallback`

That makes sense. I had originally stuck with using `ready` as part of the snippet `options` like the original PR, but realized it didn't make sense given that it wasn't being used by the snippet so I moved it to a separate parameter but didn't rename it.